### PR TITLE
Bump actions-setup-minikube

### PR DESCRIPTION
**Pull request by Tobianas**
Bump to newer version to support ubuntu 22.

JIRA: LIGHTY-113

### CODE REVIEW

**Review by PeterSuna — APPROVED** [review]

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -108,10 +108,10 @@ jobs:
           key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
           restore-keys: ${{ runner.os }}-m2
       - name: Setup Minikube
-        uses: manusa/actions-setup-minikube@v2.4.2
+        uses: manusa/actions-setup-minikube@v2.7.1
         with:
-          minikube version: 'v1.24.0'
-          kubernetes version: 'v1.22.4'
+          minikube version: 'v1.26.1'
+          kubernetes version: 'v1.25.0'
```
Comment (ihrasko) [inline .github/workflows/build.yml:114]: https://github.com/kubernetes/kubernetes/releases/tag/v1.25.4

Comment (ihrasko) [reply .github/workflows/build.yml:114]: Can we use more recent versions that are mentioned in https://github.com/manusa/actions-setup-minikube?

**Review by ihrasko — APPROVED** [review]

**Comment by Tobianas** [comment]
> Hi @Tobianas ,
> 
> Will this pr also be merged to master?

Hello @marchest,

Yes, we are currently working on it and it should be done soon.

**Comment by marchest** [comment]
Thanks @Tobianas

**Comment by Tobianas** [comment]
> Thanks @Tobianas

https://github.com/PANTHEONtech/lighty/pull/1189
should be merged soon :)

**Pull request merged**